Makefile

```diff
@@ -32,29 +32,31 @@ endif
 # Get OS architecture
 OSARCH=$(shell uname -m)
 ifeq ($(OSARCH),x86_64)
-REL_OSARCH=linux/amd64
+GOARCH?=amd64
 else ifeq ($(OSARCH),x64)
-REL_OSARCH=linux/amd64
+GOARCH?=amd64
 else ifeq ($(OSARCH),aarch64)
-REL_OSARCH=linux/arm64
+GOARCH?=arm64
 else ifeq ($(OSARCH),aarch64_be)
-REL_OSARCH=linux/arm64
+GOARCH?=arm64
 else ifeq ($(OSARCH),armv8b)
-REL_OSARCH=linux/arm64
+GOARCH?=arm64
 else ifeq ($(OSARCH),armv8l)
-REL_OSARCH=linux/arm64
+GOARCH?=arm64
 else ifeq ($(OSARCH),i386)
-REL_OSARCH=linux/x86
+GOARCH?=x86
 else ifeq ($(OSARCH),i686)
-REL_OSARCH=linux/x86
+GOARCH?=x86
 else ifeq ($(OSARCH),arm)
-REL_OSARCH=linux/arm
+GOARCH?=arm
 else
-REL_OSARCH=linux/$(OSARCH)
+GOARCH?=$(OSARCH)
 endif
 
 # Run `make images DOCKER_PLATFORMS="linux/amd64,linux/arm64" BUILDX_OUTPUT_TYPE=registry IMAGE_PREFIX=[yourregistry]` to push multi-platform
-DOCKER_PLATFORMS ?= "${REL_OSARCH}"
+DOCKER_PLATFORMS ?= "linux/${GOARCH}"
+
+GOOS ?= linux
```

Comment:
nit: it seems we donot use this at all, This env is set in go env

Reply:
Because we only compile the linux version at present, if the default value is not set to linux, then when compiling on macos, the binary in macos format will be compiled